smpte.js

```diff
@@ -135,16 +135,17 @@ function insertTOC(docMetadata) {
         continue;
 
       const headingNumber = subSection.querySelector(".heading-number");
+      const tocIgnore = subSection.querySelector(".toc-ignore")
 
-      if (! headingNumber)
+      if (!headingNumber && tocIgnore)
```

Comment:
Things that are numbered should be in the ToC.

Reply:
Yes, agreed, all items including Foreward and Bibliography need to be in the TOC. The only section with tocIgnore (to no be in the TOC) are the TOC and the header of the page.

Reply:
Updated via https://github.com/SMPTE/html-pub/pull/27/commits/1d1c2d8b590b01bba6c7d31eb9773b5271602c91